weaviate/values.yaml

```diff
@@ -79,6 +79,15 @@ service:
   clusterIP:
   annotations: {}
 
+# The service monitor defines prometheus monitoring for a set of services
+# https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#monitoring.coreos.com/v1.ServiceMonitor
+serviceMonitor:
+  enabled: false
+  port: metrics
+  path: /metrics
```

Comment:
the `port` and `path` are always the same and I do not see why they will changes so I think you can hardcode them instead of providing them in the `values.yaml`.

Reply:
agree, I've defined the port and path in the serviceMonitor resource.